### PR TITLE
Use standard JspBase.getWebappURL() and JspBase.getScriptTag() helper methods

### PR DESCRIPTION
**Pull request by labkey-adam**
#### Rationale
We have helper methods that safely and succinctly create `<script>` and `` tags.

#### Related Pull Requests
* See below

#### Changes
* Use standard JspBase.getWebappURL() and JspBase.getScriptTag() helper methods instead of hand assembling script include tags and image URLs
* Best not to HTML encode SafeToRender objects... might result in double encoding

### CODE REVIEW

File: api/src/org/labkey/api/view/GWTView.jsp [hunk 1]
```diff
@@ -34,12 +34,11 @@
...
 <div id="<%= h(bean.getModuleName()) %>-Root" class="<%=h(bean.getLoadingStyleName())%>"></div>
 <%
-String contextPath = request.getContextPath();
 String jsPath = bean.getModuleName() + "/" + bean.getModuleName() + ".nocache.js";
-String hashedPath = contextPath + "/" + jsPath + "?" + PageFlowUtil.getServerSessionHash();
+String hashedPath = "/" + jsPath + "?" + PageFlowUtil.getServerSessionHash();
 %>
 <script id="__gwt_marker_<%=h(bean.getModuleName())%>"></script>
-<script type="text/javascript" src="<%=h(hashedPath)%>"></script>
+<%=getScriptTag(hashedPath)%>
```
Comment (labkey-matthewb) [inline api/src/org/labkey/api/view/GWTView.jsp]: Shouldn't this be getScriptTag(jsPath)?  getScriptTag() adds the hash.

Comment (labkey-adam) [reply api/src/org/labkey/api/view/GWTView.jsp]: `getScriptTag()` doesn't add the hash if the path contains "nocache" (see `staticResourceUrl()`). So, passing just `jsPath` would remove the hash in this case. I suspect it doesn't matter either way... not like the GWT stuff is changing often.

Comment (labkey-adam) [reply api/src/org/labkey/api/view/GWTView.jsp]: Oops... my bad, it skips the hash if the path contains ".cache."

Comment (labkey-adam) [reply api/src/org/labkey/api/view/GWTView.jsp]: Fixed

**Review by labkey-matthewb — APPROVED** [review]

**Pull request merged**